.gitignore

```diff
@@ -4,3 +4,4 @@ node_modules
 .eslintcache
 package-lock.json
 yarn.lock
+.idea
```

Comment:
This shouldn't be in a project gitignore. This belongs to user's [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore).

Reply:
👌,
And there still some unit test error to be resolved.